test/apps/test_init_mt.c

```diff
@@ -14,10 +14,21 @@
 #include <omp.h>
 #endif
 
+#include <time.h>
+#include <sys/time.h>
+#include <stdlib.h>
+
 
 int main(int argc, char **argv)
 {
-    int count = 0;
+    time_t timeout = (argc > 1) ? atol(argv[1]) : 60;
+    int count      = 0;
+    struct timeval start;
+    struct timeval finish;
+    time_t duration;
+
+    gettimeofday(&start, NULL);
+    printf("[%ld.%06ld] .. ", start.tv_sec, start.tv_usec);
```

Comment:
print with newline, so in case test is stuck io buffer would be flushed

Reply:
added flush call

Reply:
starting test

Reply:
> starting test

updated